docs/contributing/04-prow-workflow.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Prow workflow
+---
+
+This document describes the Prow workflow that we use across all Kyma repositories. This includes basic principles, explanations, and commands that interact with [@kyma-bot](https://github.com/kyma-bot).
+
+Prow is a Kubernetes-based CI/CD system. Jobs can be triggered by various types of events and report their status to many services.
+In addition to job execution, Prow provides GitHub automation in the form of policy enforcement, ChatOps through the `/foo` style commands, and automatic PR merging.
+
+Prow enforces strict ownership policies based on the OWNERS files that reside on each organisation repository.
+The Prow bot checks if issues have the required labels assigned, automatically assigns the labels to pull requests,
```

Comment:
This sentence would fit better in the previous paragraph where you explain the Prow features.